encrypt-cluster-comms/README.md

```diff
@@ -1,5 +1,79 @@
-# Encrypt Cluster Communications
+# Encrypting Cluster Communication
+Learn how to create OpenShift secrets.
 
+## Environment Variables for Encrypting Communication
+
+Data Grid uses the following environment variables to encrypt communication:
```

Comment:
I think it'd be good to make it more explicit:

    encrypt communication between clients and servers.